.github/workflows/ci.yml

```
@@ -10,16 +10,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Cache SBT
-        uses: actions/cache@v3
+      - name: Set up JDK 17 with SBT caching
```

```suggestion
      - name: JDK with SBT caching
```
since `17`is defined below - so whenever we bump up the Java version, we don't need to also change this comment